### PR TITLE
Check all function arguments

### PR DESCRIPTION
**Pull request by NazarPonochevnyi**
Added check of all arguments of the function, because the arguments are raised where they do not need.

### CODE REVIEW

File: JavaScript/4-usage.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ application.on('smth', (data) => {
 });
 
 application.on('*', (name, data) => {
-  console.dir([name, data]);
+  if(name, data){
```
Comment (tshemsedinov) [inline JavaScript/4-usage.js:12]: Use linter before PR, add spaces for code style `if (name, data) {`

**Review by tshemsedinov — CHANGES_REQUESTED** [review]

File: JavaScript/4-usage.js (hunk 1 above)
Comment (tshemsedinov) [reply JavaScript/4-usage.js:12]: Second issue: you use two values in `if`

**Comment by tshemsedinov** [comment]
Why do we need to check all arguments?

**Comment by NazarPonochevnyi** [comment]
Yes, I already understood that I had solved the problem poorly. Give me a little time, I'll fix it differently)

**Comment by NazarPonochevnyi** [comment]
Sorry, but I don't know how fixed this problem. When I finded answer, I create new PR.

**Comment by tshemsedinov** [comment]
@NazarPonochevnyi you may reopen PR instead

**Pull request closed without merging**